domains/vijay.json

```diff
@@ -0,0 +1,11 @@
+{
+    "description": "Vijay's profile",
+    "repo": "https://github.com/hack-e-d/profile.github.io",
+    "owner": {
+        "username": "hack-e-d",
+        "email": "pvvijayraj1998@gmail.com",
+    },
+    "record": {
+        "CNAME": "hack-e-d.github.io"

```

Comment:
That shows invalid GitHub site, and the repository is not found either. Maintainers cannot verify the site's content without those.

Reply:
Only if I add the /profile dose the site load so how do I mention the link ?

Reply:
> Only if I add the /profile dose the site load so how do I mention the link ?

You can change the name of the repository to "<YOUR_USERNAME>.github.io" but it isn't required.
https://hack-e-d.github.io/profile/ is your website.

Reply:
Yes this is it